lib/airbrake/rack/user.rb

```diff
@@ -50,7 +50,12 @@ def as_json
       private
 
       def try_to_get(key)
-        String(@user.__send__(key)) if @user.respond_to?(key)
+        String(@user.__send__(key)) if can_call_method?(key)
```

Comment:
My preference would be the following implementation, however I appreciate the feedback from the previous reviewer. ⛳️
 
```rb
def try_to_get(key)
  return unless @user.respond_to?(key)
  return if (-1..0).include?(@user.method(key).arity)
  String(@user.__send__(key))
end